hedera-node/src/main/java/com/hedera/services/utils/MiscUtils.java

```diff
@@ -217,6 +217,12 @@ private MiscUtils() {
 			NetworkGetExecutionTime
 	);
 
+	private static final Set<HederaFunctionality> CONSENSUS_THROTTLED_FUNCTIONS = EnumSet.of(
+			ContractCallLocal,
```

Comment:
I think that we need to remove `ContractCallLocal` since its not consensus throttled